graphdatascience/tests/unit/test_graph_ops.py

```diff
@@ -142,8 +142,7 @@ def test_graph_export_csv(runner: CollectingQueryRunner, gds: GraphDataScience)
 def test_graph_streamNodeProperty(runner: CollectingQueryRunner, gds: GraphDataScience) -> None:
     G, _ = gds.graph.project("g", "*", "*")
 
-    with pytest.warns(DeprecationWarning):
-        gds.graph.streamNodeProperty(G, "dummyProp", concurrency=2)
+    gds.graph.streamNodeProperty(G, "dummyProp", concurrency=2)
```

Comment:
Its not undeprecated, but the warning comes from the server now instead of the client.
So in a unit-test, we cannot assert on this warning anymore only in integration tests

Reply:
Ooooh unit testing, I get it. Cool!